### DISCUSSION TITLE
Has anyone found any way to properly ignore fusion 360?

### DISCUSSION THREAD

**Discussion by Vib1240n** [asmvik/yabai#2471] [Q&A]
I have this in my config - 
```
  60   │ yabai -m rule --add app="^Fusion$" title!=".*\[(.*)\].*" manage=off
  61   │ yabai -m rule --add app="^Autodesk Fusion 360$" manage=off
```
but this doesn't really do anything when it comes to ignoring Fusion 360. Anyone got any better ideas?

**Comment by alchemistake** [comment]
https://github.com/koekeishiya/yabai/issues/2410#issuecomment-2820748082 i found a way
